Compiler/Util/Flags.mo

```diff
@@ -364,7 +364,7 @@ constant DebugFlag INFO_XML_OPERATIONS = DEBUG_FLAG(96, "infoXmlOperations", fal
   Util.gettext("Enables output of the operations in the _info.xml file when translating models."));
 constant DebugFlag HPCOM_DUMP = DEBUG_FLAG(97, "hpcomDump", false,
   Util.gettext("Dumps additional information on the parallel execution with hpcom."));
-constant DebugFlag RESOLVE_LOOPS = DEBUG_FLAG(98, "resolveLoops", true,
+constant DebugFlag RESOLVE_LOOPS = DEBUG_FLAG(98, "resolveLoops", false,
```

Comment:
For testing purpose, you can also run Hudson on your branches without creating pull requests.

It is possible to deactivate all backend modules without special flags, e.g. `preOptModules-=resolveLoops`.